style.css

```diff
@@ -1 +1,347 @@
+
+  display: grid;
+  grid-gap: 20px;
+  grid-template-areas: 
+  "main-img main-paragrafo" 
+  "main-img main-paragrafo";
+
+}
+
+.main-img{
+  background-color: #feeafa;
+  grid-area: main-img;
+  min-height: 500px;
+  background-size: cover;
+  background-position: left;
+  padding: 3rem;
+  display: flex;
+  flex-direction: column;
+  align-self: start;
+  justify-content: left;
+  margin: auto;
+  }
+.main-paragrafo{
+  background-color: #feeafa;
+  grid-area: main-paragrafo;
+  max-height: 500px;
+  background-size: cover;
+  background-position: right;
+  padding: 3rem;
+  display: flex;
+  align-self: right;
+  flex-direction: column;
+  justify-content: right;
+
+}
+
+.main-paragrafo h1{
+  font-size: 4rem;
+  justify-content: center;
+  text-align: center;
+  padding: 4px;
+  text-align: center;
+}
+
+.main-paragrafo p{
+  font-size: 1.5rem;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.portfolio{
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+background-color: #feeafa;
+padding: 3rem;
+
+}
+
+.portfolio .images1{
+  width: 100%;
+  box-shadow: var(--shadow);
+}
+.portfolio .images2{
+  width: 100%;
+  box-shadow: var(--shadow)
+ 
+}
+.portfolio .images3{
+  width: 100%;
+  box-shadow: var(--shadow)
+}
+.portfolio .images4{
+  width: 100%;
+  box-shadow: var(--shadow)
+}
+
+.nav{
+  text-decoration: none;
+  
+}
+/* footer*/
+.footer{
+
+  display: grid;
+  text-decoration: none;
+  text-align: center;
+  color: var(--ligth);
+  font-size: 1.2rem;
+  box-shadow: var(--shadow);
+  background-color: #feeafa;
+  margin: 2em;
+padding: 1em;
+}
+
+.footer h1{
+  color: var(--dark);
+  padding-top: 1em;
+  justify-content: center;
+  padding-bottom: 0.5em;
+}
+
+.footer P{
+  padding: 2em;
+  justify-content: center;
+  color: var(--dark);
+  padding-top: 0.5em;
+  cursor: pointer;
+}
+
+
+.footer{
+  grid-template-areas: "footer";
+  margin: 1px;;
+}
+ 
+ /*   
+ 
+  margin-right: 500px;
+  margin-left: 500px;
+  margin-top: 100px;
+  margin-bottom: 300px;
+  justify-content: center;
+  text-align: center;
+  align-self: center;
+
+}
+*/
+
+.box-social{
+  display:flex;
+  grid-gap: 30px;
+ grid-template-areas:
+  "card-social1 card-social2 card-social3";
+  margin: 1px;
+  justify-content: center;
+  padding-bottom: 1px;
+}
+
+.card-social1{
+  grid-area: card-social1;
+  width: 40px;
+  cursor: pointer;
+}
+
+.card-social2{
+  grid-area: card-social2;
+  width: 40px;
+  cursor: pointer;
+}
+
+.card-social3{
+  grid-area: .card-social3;
+  width: 40px;
+  cursor: pointer;
+}
+.card-social{
+  border: 5px; solid #b1b4b7;
+  border-radius: 10px;
+  padding: 2em ;
+
+}
+
+
+/*  imag*/
+
+.logo{
+width: 200px; 
+}
+
+
+
+.images1{
+width: 300px;
+
+}
+
+.images2{
+width: 300px;
+height: auto;}
+
+.images3{
+ width: 300px;
+}
+ .images4{
+ width: 300px;
+height: auto;}
+
+.main-cake{
+width: 750px;
+}
+
+/*media queries*/
+
+@media ( max-width: 540px) {
```

Comment:
I actually quite like the way the buttons go on smaller screens, it's not what the design asked for but it's still really nice in it's own way, could have done with a bit more work to add a middle breakpoint and make the smallest breakpoint look like the mobile veiw